models/webhooks/store_product_updated.yml

```diff
@@ -15,6 +15,15 @@ properties:
             type: string
           id:
             type: integer
+          attributes:
+            type: string

```

Comment:
should be array of strings
```suggestion
            type: array

```

Reply:
also we might need to mentioned the exact set of allowed values here, like `name`, `description`, etc